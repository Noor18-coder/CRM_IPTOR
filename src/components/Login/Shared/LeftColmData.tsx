import React from 'react';
import { Image } from 'react-bootstrap';
import logo from '../../../assets/images/iptor-logo-white.svg';

const LeftColmData = () => {
  return (
    <div className="login-lft-cont-container">
      <Image className="login-cont-logo" src={logo} alt="Iptor" title="Iptor" />
      <h1 className="para">A milestone in the Iptor transformation journey</h1>
      <p className="version-details">
        Version 11 is the key platform for distribution and publishing businesses to turn digital disruption into digital opportunity.
      </p>
    </div>
  );
};

export default LeftColmData;
