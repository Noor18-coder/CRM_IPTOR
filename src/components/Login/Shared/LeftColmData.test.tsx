import renderer from 'react-test-renderer';
import LeftColmData from './LeftColmData';

describe('[Login Shared] LeftColmData', () => {
  it('should renders correctly', () => {
    const tree = renderer.create(<LeftColmData />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
