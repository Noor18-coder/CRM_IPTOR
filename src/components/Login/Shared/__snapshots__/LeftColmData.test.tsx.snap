// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Login Shared] LeftColmData should renders correctly 1`] = `
<div
  className="login-lft-cont-container"
>
  <img
    alt="Iptor"
    className="login-cont-logo"
    src="iptor-logo-white.svg"
    title="Iptor"
  />
  <h1
    className="para"
  >
    A milestone in the Iptor transformation journey
  </h1>
  <p
    className="version-details"
  >
    Version 11 is the key platform for distribution and publishing businesses to turn digital disruption into digital opportunity.
  </p>
</div>
`;
